libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java

```diff
@@ -143,7 +147,41 @@ public UserAccount(String authToken, String refreshToken,
 			String orgId, String userId, String username, String accountName,
 			String clientId, String communityId, String communityUrl,
             String firstName, String lastName, String displayName, String email, String photoUrl,
-            String thumbnailUrl ) {
+            String thumbnailUrl) {
+		this(authToken, refreshToken, loginServer, idUrl, instanceServer, orgId, userId, username,
+				accountName, clientId, communityId, communityUrl, firstName, lastName, displayName,
+				email, photoUrl, thumbnailUrl, null);
+	}
+
+	/**
+	 * Parameterized constructor.
+	 *
+	 * @param authToken Auth token.
+	 * @param refreshToken Refresh token.
+	 * @param loginServer Login server.
+	 * @param idUrl Identity URL.
+	 * @param instanceServer Instance server.
+	 * @param orgId Org ID.
+	 * @param userId User ID.
+	 * @param username Username.
+	 * @param accountName Account name.
+	 * @param clientId Client ID.
+	 * @param communityId Community ID.
+	 * @param communityUrl Community URL.
+	 * @param firstName First Name.
+	 * @param lastName Last Name.
+	 * @param displayName Display Name.
+	 * @param email Email.
+	 * @param photoUrl Photo URL.
+	 * @param thumbnailUrl Thumbnail URL.
+	 * @param customIdentityValues Custom identity values.
+	 */
+	public UserAccount(String authToken, String refreshToken,
```

Comment:
Is there any plan to move to a builder pattern for these constructors? Might reduce some of your overhead.